### PR TITLE
Use better description in pos_payment.js

### PR DESCRIPTION
**Pull request by mahsem**
Use Change Amount instead of Change and To Be Paid in pos_payment.js and be consistent with other strings

### CODE REVIEW

File: erpnext/selling/page/point_of_sale/pos_payment.js [hunk 1]
```diff
@@ -584,7 +584,7 @@ erpnext.PointOfSale.Payment = class {
 		const remaining = grand_total - doc.paid_amount;
 		const change = doc.change_amount || remaining <= 0 ? -1 * remaining : undefined;
 		const currency = doc.currency;
-		const label = change ? __("Change") : __("To Be Paid");
+		const label = change ? __("Change Amount") : __("Change Amount");
```
Comment (s-aga-r) [inline erpnext/selling/page/point_of_sale/pos_payment.js]: Both the true and false branches of the ternary operator are the same.

Comment (mahsem) [reply erpnext/selling/page/point_of_sale/pos_payment.js]: Yes but logically thinking from sales person view it is right

Comment (s-aga-r) [reply erpnext/selling/page/point_of_sale/pos_payment.js]: > Yes but logically thinking from sales person view it is right

That's okay, I was talking about the way you fix it. It should be `const label = __("Change Amount");` not `const label = change ? __("Change Amount") : __("Change Amount");`

Comment (mahsem) [reply erpnext/selling/page/point_of_sale/pos_payment.js]: Will do new PR

Comment (s-aga-r) [reply erpnext/selling/page/point_of_sale/pos_payment.js]: Make the change in this PR only?

**Review by mahsem — COMMENTED** [review]
Fixed

**Pull request merged**